index.html

```diff
@@ -1412,6 +1412,104 @@ <h3>Biometrics</h3>
     </section>
   </section>
 
+  <section class="informative">
+    <h2>Environmental and Ethical Considerations</h2>
+
+    <p>
+      The following section details certain key areas of focus when implementing a 
+      new DID method, or implementing a solution that incorporates DIDs.
+    </p>
+
+    <p>
+      The <a 
+      href="https://www.w3.org/2001/tag/doc/ethical-web-principles/">W3C TAG
+      Ethical Web Principles</a> document is a reliable guide for making assessments
+      and weighing ethical considerations of various technologies.
```

Comment:
```suggestion
      Ethical Web Principles</a> document is a helpful guide
      when weighing ethical considerations of various technologies.
```
@jandrieu -- Satisfactory?